cfgov/unprocessed/js/modules/behavior/FlyoutMenu.js

```diff
@@ -1,9 +1,14 @@
 // Required modules.
-const BaseTransition = require( '../../modules/transition/BaseTransition' );
-const behavior = require( '../../modules/util/behavior' );
-const breakpointState = require( '../../modules/util/breakpoint-state' );
-const EventObserver = require( '../../modules/util/EventObserver' );
-const standardType = require( '../../modules/util/standard-type' );
+import BaseTransition from '../../modules/transition/BaseTransition';
+import { checkBehaviorDom } from '../../modules/util/behavior';
+import * as breakpointState from '../../modules/util/breakpoint-state';
```

Comment:
Mostly a nit, but it would be more concise if throughout this PR to move from collecting exports in `{ ... }` then importing them with `import * as...` to exporting objects as `default`, eg:
```
export default {
  get,
  isInDesktop
}
```

Then importing with `import breakpointState from ...`

Reply:
Unfortunately default exports break IE10:

<img width="260" alt="screen shot 2018-11-27 at 9 51 29 am" src="https://user-images.githubusercontent.com/704760/49089685-16363680-f22a-11e8-8505-b522a7938662.png">

We may need a polyfill. Dirty little secret: we don't automatically polyfill with babel-polyfill currently as it can be overly aggressive.

Reply:
Interesting! Thanks for the info, definitely something to look into further.